src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -45,23 +43,24 @@ export default class ReplyField extends React.Component<Props, State> {
         value: '',
     };
 
-    constructor(props: Props) {
```

Comment:
Does this still have to be a class?

Reply:
We still need to rely on the React lifecycles `componentDidUpdate` and `componentWillUnmount`